### ISSUE TITLE
Improve docs on bloom

### ISSUE THREAD

**Issue by cBournhonesque** [bevyengine/bevy#14440]
## How can Bevy's documentation be improved?

I was trying to get bloom working and I couldn't understand what I need to do, even after reading the docs and going through the examples.

It looks like bloom needs to use color values higher than 1.0 in `Color::LinearSRGB` space to be enabled?
An easy way is to do something like `Color::from(color.to_linear() * 5.0)`

**Comment by alice-i-cecile** [comment]
Correct, we're using the LinearRgba space in a physical sense that represents quantity of light. They don't actually have to be greater than 1.0 I believe, but generally you'll want them to be.
